.github/workflows/validate.yml

```diff
@@ -56,3 +56,11 @@ jobs:
         shell: 'script --return --quiet --command "bash {0}"'
         run: |
           make -f docker.Makefile ${{ matrix.target }}
+      -
+        name: Validate generated # check that checked-in and generated files match
```

Comment:
I guess we need to skip the yaml docs, because we don't commit those into source control currently (we do commit the markdown files though)